applications/nfc/scenes/nfc_scene_mf_classic_keys_list.c

```diff
@@ -59,6 +69,12 @@ bool nfc_scene_mf_classic_keys_list_on_event(void* context, SceneManagerEvent ev
 
 void nfc_scene_mf_classic_keys_list_on_exit(void* context) {
     Nfc* nfc = context;
-    // TODO: Find some way to clear all those strings from submenu.
+
+    MfClassicUserKeys_it_t it;
```

Comment:
Memory leak was mostly fixed with latest commit, however not completely; I suspect it has to do with this array iterator.